changelog.d/pr-7550

```diff
@@ -0,0 +1,10 @@
+synopsis: Add --open flag to `cabal haddock`
+packages: cabal-install
+prs: #7550
+issues: #5115
+
+description: {
+
+- Add --open flag to `haddock` subcommand that attempts to open the generated documentation in a browser.
```

Comment:
since it's a single item it can go in the synopsis instead, otherwise on the changelog it will appear as something like:

> * Add --open flag to `cabal haddock`
>   * Add --open flag to `haddock` subcommand that attempts to open the generated documentation in a browser.